packages/shared/src/components/fields/MarkdownInput/index.tsx

```diff
@@ -276,6 +276,7 @@ function MarkdownInput(
               )}
               icon={icon}
               onClick={() => uploadRef?.current?.click()}
+              type="button"
```

Comment:
This fixed an issue that upon clicking on any icon on the markdown bar it submitted the form